pkg/util/ecs/metadata/detection.go

```diff
@@ -106,7 +106,6 @@ func testURLs(urls []string, timeout time.Duration) string {
 		}
 		var resp v1.Commands
 		if err := json.NewDecoder(r.Body).Decode(&resp); err != nil {
-			fmt.Printf("decode err: %s\n", err)
```

Comment:
shouldn't we replace it with a log message?

Reply:
we don't log anything in the entire `detection` logic, so I'm assuming we don't need to here either (and I don't know if it'd become spammy if we did).

I can change this to a debug log though

Reply:
Adding a debug log could be useful yes, recently the ecs api broke the backward compatibility of one of its endpoints by returning a json with the same content but with different format, so we'll never know 🤷‍♂️ (cc @xornivore )

Reply:
👍 addressed

Reply:
thank you!